apps/web/modules/event-types/views/event-types-listing-view.tsx

```diff
@@ -884,41 +885,50 @@ const Main = ({
     rawData.eventTypeGroups.length === 1;
 
   const data = denormalizePayload(rawData);
-
   return (
     <>
       {data.eventTypeGroups.length > 1 || isFilteredByOnlyOneItem ? (
         <>
           {isMobile ? (
             <MobileTeamsTab eventTypeGroups={data.eventTypeGroups} />
           ) : (
-            data.eventTypeGroups.map((group, index: number) => (
-              <div
-                className="mt-4 flex flex-col"
-                data-testid={`slug-${group.profile.slug}`}
-                key={group.profile.slug}>
-                <EventTypeListHeading
-                  profile={group.profile}
-                  membershipCount={group.metadata.membershipCount}
-                  teamId={group.teamId}
-                  bookerUrl={group.bookerUrl}
-                />
+            data.eventTypeGroups.map((group, index: number) => {
+              const userHasManagedEventType = group.eventTypes.find(
+                (event) => event.metadata?.managedEventConfig
+              );
+              if (group.profile.eventTypesLockedByOrg && !userHasManagedEventType) return null;
```

Comment:
If locked and no managed event types - we dont need to show to show this "group"

Reply:
what about the hidden event types? shouldn't we still show them

Reply:
we should also lock the toggle of the hidden event types

Reply:
Good spot! I always had a managed event in there when testing :P